### PR TITLE
update neard 1.31.0-rc.2

### PR DESCRIPTION
**Pull request by yanganto**

### CODE REVIEW

File: nix/pkgs/neard/unstable.nix [hunk 1]
```diff
@@ -4,7 +4,7 @@ let
   generic = pkgs.callPackage ./generic.nix { };
 in
 generic {
-  version = "1.31.0-rc.1";
-  sha256 = "sha256-H5L2CUpou1lIqX0IkPNoM0IyUiL4Y2sIDFCqjdB41RU=";
-  cargoSha256 = "sha256-/Q8Vf8mWrEaT+KmCtoSWwrY9/sAYIS94UuQI/xdLpMU=";
+  version = "1.31.0-rc.2";
+  sha256 = "sha256-5TldO83qosAGZmcawqzlZOJ+dWMP9V1f4byD0lLKYE0=";
+  cargoSha256 = "sha256-T+HfPD2NR2mZ3HxZ95i1iIvB8Zy+KHcXeTuAlIm0Ci8=";
```
Comment (Mic92) [inline nix/pkgs/neard/unstable.nix:9]: Is this cargoSha256 hash correct?

Comment (Mic92) [reply nix/pkgs/neard/unstable.nix:9]: CI says otherwise.

Comment (yanganto) [reply nix/pkgs/neard/unstable.nix]: It is the same one. 
https://github.com/kuutamolabs/near-staking-knd/pull/444/checks?check_run_id=10599757189

Comment (Mic92) [reply nix/pkgs/neard/unstable.nix:9]: Ok. same as here: https://github.com/kuutamolabs/near-staking-knd/pull/446/files
I think the cargo.lock file wants to update itself for some reason, we may need `cargoPatches` in nix.

Comment (yanganto) [reply nix/pkgs/neard/unstable.nix:9]: Got it.  Thanks.

**Comment by Mic92** [comment]
Is the unitest error legit?

**Comment by yanganto** [comment]
> Is the unitest error legit?

There should be metrics when maintenance shutdown set.  I will check this manually first.

**Comment by yanganto** [comment]
The test case passed locally and quickly.  I will add additional 30s to wait for the metrics for this PR.

**Pull request merged**